lib/src/api/engine/local/mod.rs

```diff
@@ -260,6 +260,28 @@ pub struct IndxDb;
 #[derive(Debug)]
 pub struct TiKv;
 
+/// Sled database
+///
+/// # Examples
+///
+/// Instantiating a Sled instance
+///
+/// ```no_run
+/// # #[tokio::main]
+/// # async fn main() -> surrealdb::Result<()> {
+/// use surrealdb::opt::Strict;
+/// use surrealdb::Surreal;
+/// use surrealdb::engine::local::Sled;
+///
+/// let db = Surreal::new::<Sled>(("sled.db")).await?;
```

Comment:
Did you mean

```rust
let db = Surreal::new::<Sled>(("sled.db", Strict)).await?;
```

or just

```rust
let db = Surreal::new::<Sled>("sled.db").await?;
```

?

Reply:
I'm assuming it's the former because of the `surrealdb::opt::Strict` import.

Reply:
Absolutely right, in fact I mixed up the strict and non-strict examples. Fixing it now